tox.ini

```diff
@@ -11,7 +11,7 @@ deps =
     pyserial~=3.0
 
 commands =
-    pytest
+    pytest {posargs}
```

Comment:
What's this?

Reply:
Allows additional args to be passed through from tox. Needed to just run a single test.